common-rest/src/main/resources/uniprotkb-query.config

```diff
@@ -14,6 +14,9 @@ annotation_score
 # QUERY-FIELDS
 accession,content,ec,id_default,id_inactive,keyword,virus_host_name,virus_host_id,taxonomy_name,taxonomy_id,protgene_default,organelle,database,lit_title,lit_author,lit_pubmed,lit_journal,transposon,strain,plasmid,tissue,scope,xref,chebi,rhea,family
```

Comment:
Can you please add accession boost
accession:{query}^5.0

Reply:
done